### PR TITLE
Release/1.4.1

### PR DESCRIPTION
**Pull request by bryanculver**
Request to release 1.4.1

### CODE REVIEW

File: nautobot_secrets_providers/providers/hashicorp.py [hunk 1]
```diff
@@ -151,11 +151,14 @@ def get_client(cls, secret=None):
                 elif auth_method == "aws":
                     session = boto3.Session()
                     aws_creds = session.get_credentials()
+                    aws_region = session.region_name
+                    if aws_region is None:
+                        aws_region = "us-east-1"
```
Comment (HanlinMiao) [inline nautobot_secrets_providers/providers/hashicorp.py:156]: Nit pick pls do not hate me.
```suggestion
                    aws_region = getattr(session, "region_name", "us-east-1")
```

**Review by HanlinMiao — APPROVED** [review]

File: nautobot_secrets_providers/providers/hashicorp.py (hunk 1 above)
Comment (glennmatthews) [reply nautobot_secrets_providers/providers/hashicorp.py:156]: That's functionally different though - `session.region_name` is `None`, not nonexistent.

Comment (HanlinMiao) [reply nautobot_secrets_providers/providers/hashicorp.py:156]: Could we do this then?
```suggestion
                    aws_region = session.region_name or "us-east-1"
```

Comment (bryanculver) [reply nautobot_secrets_providers/providers/hashicorp.py:156]: @glennmatthews Thoughts on the above or is the current good enough?

Comment (glennmatthews) [reply nautobot_secrets_providers/providers/hashicorp.py:156]: Either approach is fine by me.

**Pull request merged**